gax/src/main/java/com/google/api/gax/paging/Page.java

```diff
@@ -52,8 +52,9 @@
   Page<ResourceT> getNextPage();
 
   /**
-   * Returns an iterable over all elements, starting from this page. Elements of the list are
-   * retrieved lazily using the underlying API.
+   * Returns an iterable that traverses all of the elements of the underlying data source. The
+   * data is fetched lazily page by page, where each page contains multiple elements. A new page is
```

Comment:
probably more correct would be "may contain multiple elements", since pages of size one should also be allowed (same for teh other doc).